app/controllers/api/v4/drug_stocks_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V4::DrugStocksController < APIController
+  def index
+    if for_end_of_month.nil?
+      render status: 400, json: {errors: ["valid `date` must be provided - eg. 2021-10-29"]}
```

Comment:
Shall we also specify the date format we expect in this (already very helpful) error message?